tests/tests/test_basic_integration.py

```diff
@@ -139,12 +139,50 @@ def deployment_callback():
                 template = run(catcmd)
                 wait_count = 0
 
+    @pytest.mark.timeout(1000)
```

Comment:
Why the timeout?

Reply:
If the client doesn't work, then the test will be stuck.

Reply:
No, it's ok, I'd rather have a long timeout, given our history with crappy droplets. I was just curious!